extensions/wikia/WikiFactory/Close/gcs_bucket_remover.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use Wikia\Logger\Loggable;
+
+class GcsBucketRemover {
```

Comment:
If this class is just needed for the `remove` function, I think we can move `remove` to the maintenance script. I originally thought there would be more use for a storage wrapper.

Reply:
I extracted it because we have 2 maintenance scripts removing bucket - maintenance and close single wiki - I'm not sure if both are used, but they were ported to UCP: https://github.com/Wikia/unified-platform/pull/130/files